package.json

```diff
@@ -33,20 +33,21 @@
     "contrib/"
   ],
   "devDependencies": {
-    "@esotericsoftware/spine-core": "4.1.36",
     "@playcanvas/eslint-config": "^1.5.0",
+    "@rollup/plugin-alias": "^5.0.0",
+    "@rollup/plugin-commonjs": "^25.0.4",
+    "@rollup/plugin-node-resolve": "^15.2.1",
+    "@rollup/plugin-terser": "^0.4.3",
     "esbuild": "^0.19.0",
     "eslint": "^8.46.0",
-    "grunt": "^1.6.1",
-    "grunt-contrib-concat": "^2.1.0",
-    "grunt-contrib-uglify": "^5.2.2",
     "http-server": "^14.1.1",
-    "semver": "^7.5.4"
+    "rollup": "^3.28.1",
+    "semver": "^7.5.4",
+    "spine40": "npm:@esotericsoftware/spine-core@4.0",
```

Comment:
OK, last change request...just remove this line and I'll merge.

Reply:
The alias ?  Ok.  Guess it makes the 4.0 PR clearer.

Reply:
Yeah, exactly. It'll keep that PR more 'atomic'. 👍

Reply:
Thank you. Completed in: 014a17c